demos/browser/ChimeMeetings.js

```diff
@@ -0,0 +1,49 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const AWS = require('aws-sdk');
+
+module.exports = class ChimeMeetings {
+
+    // Create an AWS SDK Chime object. Region 'us-east-1' is currently required.
+    // Use the MediaRegion property below in CreateMeeting to select the region
+    // the meeting is hosted in.
+    
+
+    constructor(currentRegion, controlRegion, endpoint, useChimeGlobal){
```

Comment:
1. Can we remove the `currentRegion` arg?
2. Consider `useChimeNamespace` instead of `useChimeGlobal` since "global" isn't referenced in any public docs

Reply:
updated